### PR TITLE
Add: Adding content to page load strategy

### PR DESCRIPTION
**Pull request by Harsha509**
### Description
Adding info on document.readyState and pageLoadStrategy behaviour for SPA.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/page_loading_strategy.de.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ it follows the _normal_ pageLoadStrategy.
 It is always recommended to stop downloading additional 
 resources (like images, css, js) when the page loading takes lot of time.
 
+`document.readyState` property of a document describes the loading state of current document.
```
Comment (diemol) [inline docs_source_files/content/webdriver/page_loading_strategy.de.md]:
```suggestion
The `document.readyState` property describes the loading state of the current document.
```

File: docs_source_files/content/webdriver/page_loading_strategy.de.md [hunk 2]
```diff
@@ -9,6 +9,17 @@ it follows the _normal_ pageLoadStrategy.
...
+By default, webdriver will hold off on responding to a `driver.get()` (or) `driver.navigate().to()`
```
Comment (diemol) [inline docs_source_files/content/webdriver/page_loading_strategy.de.md]:
```suggestion
By default, WebDriver will hold off on responding to a `driver.get()` (or) `driver.navigate().to()` 
```

File: docs_source_files/content/webdriver/page_loading_strategy.de.md [hunk 3]
```diff
@@ -9,6 +9,17 @@ it follows the _normal_ pageLoadStrategy.
 It is always recommended to stop downloading additional 
 resources (like images, css, js) when the page loading takes lot of time.
 
+`document.readyState` property of a document describes the loading state of current document.
+By default, webdriver will hold off on responding to a `driver.get()` (or) `driver.navigate().to()` 
+call until the document ready state is `complete`
+
+### Note: 
+In SPA (Single Page Applications) as once they reach the document ready state of complete,
```
Comment (diemol) [inline docs_source_files/content/webdriver/page_loading_strategy.de.md]: This paragraph is not clear, I don't understand what is trying to say.

Could you maybe describe what you want to achieve and we could work on it together? If you have any links where the text is based on, that'd be helpful.

File: docs_source_files/content/webdriver/page_loading_strategy.de.md [hunk 4]
```diff
@@ -247,3 +258,8 @@ fun main() {
 }
   {{< / code-panel >}}
 {{< / code-tab >}}
+
+### Note: 
+For most of the forms that involves testing in a browser, one should always use
```
Comment (diemol) [inline docs_source_files/content/webdriver/page_loading_strategy.de.md]: `For most of the forms that involves testing in a browser`
Same comment as above but for this sentence.

Comment (Harsha509) [reply docs_source_files/content/webdriver/page_loading_strategy.de.md]: hi @diemol, Removed the note for now. Will Cover in a separate PR with solid content.

File: docs_source_files/content/webdriver/page_loading_strategy.de.md (hunk 3 above)
Comment (Harsha509) [reply docs_source_files/content/webdriver/page_loading_strategy.de.md]: Hi @diemol, Updated content. Can you please review it.

File: docs_source_files/content/webdriver/page_loading_strategy.de.md (hunk 2 above)
Comment (Harsha509) [reply docs_source_files/content/webdriver/page_loading_strategy.de.md]: Updated the changes

File: docs_source_files/content/webdriver/page_loading_strategy.de.md (hunk 1 above)
Comment (diemol) [reply docs_source_files/content/webdriver/page_loading_strategy.de.md]: Could you please have a look at this suggestion @Harsha509?

Comment (Harsha509) [reply docs_source_files/content/webdriver/page_loading_strategy.de.md]: Apologies @diemol. Updated just now

**Pull request merged**